fe/fe-core/src/main/java/com/starrocks/persist/CreateInsertOverwriteJobLog.java

```diff
@@ -37,7 +37,11 @@ public class CreateInsertOverwriteJobLog implements Writable {
     @SerializedName(value = "targetPartitionIds")
     private List<Long> targetPartitionIds;
 
-    public CreateInsertOverwriteJobLog(long jobId, long dbId, long tableId, List<Long> targetPartitionIds) {
+    @SerializedName(value = "dynamicOverwrite")
+    private boolean dynamicOverwrite = false;
+
```

Comment:
Better to add an empty constructor.
The default value will be null after deserialization without empty constructor.